src/components/search/searchModal.js

```diff
@@ -27,20 +27,22 @@ const SearchModal = ({ isOpen, setIsOpen, onClose }) => {
   return (
     <Modal isOpen={isOpen} onClose={onClose} initialFocus={searchFieldRef}>
       <div className="su-max-w-1000 su-mx-auto">
-        <Heading
-          font="serif"
-          size={2}
-          level={2}
-          className="su-text-white su-text-center"
-        >
-          Hello, what can we help you find today?
-        </Heading>
+        <div className="su-mb-108">
```

Comment:
```suggestion
        <div className="su-rs-mb-4">
```
Please use responsive-spacing su-rs-______ classes for these larger spacings